terminus-ui/input/src/input.component.ts

```diff
@@ -1528,7 +1528,7 @@ export class TsInputComponent implements
         let stringifiedDate: string | undefined;
 
         if (this.mask === 'date') {
-          stringifiedDate = this.isValidDateString(value) ? formatDate(value, 'MM-dd-yyyy') : value;
+          stringifiedDate = this.isValidDateString(value) ? new TsDatePipe().transform(value, 'short') : value;
```

Comment:
We should be relying on the Angular injector for these. If you add the `TsPipesModule` to the imports for this module, then you can just inject the pipe via the constructor:

```
constructor(private datePipe: TsDatePipe) {}
```